fk7263/src/test/java/se/inera/intyg/common/fk7263/model/converter/certificate/question/QuestionPatientensArbetsformogaBedomsTest.java

```diff
@@ -0,0 +1,193 @@
+        }
+    }
+
+    @Nested
+    class IncludeConfigViewListTests extends ConfigViewListTest {
+
+        @Override
+        protected CertificateDataElement getElement() {
+            return QuestionPatientensArbetsformogaBedoms.toCertificate(null, null, null,
+                null, 0, messagesProvider);
+        }
+
+        @Override
+        protected String getDescriptionId() {
+            return null;
+        }
+
+        @Override
+        protected CertificateMessagesProvider getMessageProviderMock() {
+            return messagesProvider;
+        }
+
+        @Override
+        protected String getMessageId() {
+            return null;
+        }
+    }
+
+    @Nested
+    class IncludePatientensArbetsformogaBedomsValueTest {
+
+        @Test
+        void shouldDisplayAllValues() {
+            when(messagesProvider.get(any())).thenReturn("test string");
+
+            final var nuvarandeArbetsuppgifter = "test";
+            final var expectedValue = List.of(
+                CertificateDataValueViewText.builder()
+                    .text("test string" + " - " + nuvarandeArbetsuppgifter)
+                    .build(),
+                CertificateDataValueViewText.builder()
+                    .text("test string")
+                    .build(),
+                CertificateDataValueViewText.builder()
+                    .text("test string")
+                    .build()
+            );
+
+            final var element = QuestionPatientensArbetsformogaBedoms.toCertificate(true,
+                true, true, nuvarandeArbetsuppgifter, 0, messagesProvider);
+
+            final var actualValue = (CertificateDataValueViewList) element.getValue();
+
+            assertEquals(expectedValue, actualValue.getList());
+        }
+
+        @Test
+        void shouldNotDisplayValues() {
+
+            final var expectedValue = Collections.emptyList();
+
+            final var element = QuestionPatientensArbetsformogaBedoms.toCertificate(false,
+                false, false, null, 0, messagesProvider);
+
+            final var actualValue = (CertificateDataValueViewList) element.getValue();
+
+            assertEquals(expectedValue, actualValue.getList());
+        }
+
+        @Test
+        void shouldDisplayFirstValue() {
+            when(messagesProvider.get(any())).thenReturn("test string");
+
+            final var expectedValue = List.of(
+                CertificateDataValueViewText.builder()
+                    .text("test string")
+                    .build()
+            );
+
+            final var element = QuestionPatientensArbetsformogaBedoms.toCertificate(true,
+                false, false, null, 0, messagesProvider);
+
+            final var actualValue = (CertificateDataValueViewList) element.getValue();
+
+            assertEquals(expectedValue, actualValue.getList());
+        }
+
+        @Test
+        void shouldDisplayThirdValue() {
+            when(messagesProvider.get(any())).thenReturn("test string");
```

Comment:
Perhaps the names of this test and and the following one should be the other way around?

Reply:
I agree, done!